### PR TITLE
Added helpful links for Contribution

### LINKED ISSUE #19: Need to add a list of links of the World Air Quality Index's API website/documentation to the CONTRIBUTING file
Ozone uses the WAQI's API under the hood to fetch its data. Therefore, it's important that contributors can easily find all the information they need about the API in order to make good quality contributions. 

Add a section in the CONTRIBUTING.md file with a list of links to any page that has information that might be important for contributors to know. Also in a single short phrase, describe the page that the link leads to (would help people know which one they need). 

Here's a couple of links to get you started:

WAQI website: https://waqi.info
API overview: https://aqicn.org/api/

### PR DESCRIPTION
**Pull request by Tomiiwa**
Fixes #19

### CODE REVIEW

**Comment by Sam-damn** [comment]
@Tomiiwa i will review this now

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -157,6 +159,15 @@ Hold my beer, I got this
 
 
 
+## World Air Quality Index's API
+
+Ozone uses the World Air Quality Index's API under the hood to fetch its data. We have included some links to information about the API to help during contribution.
+
+* Find information on the World Air Quality Index website [here](https://waqi.info/)
+* Find information on API used by Air Quality Programs [here](https://aqicn.org/api/) 
+
```
Comment (Sam-damn) [inline CONTRIBUTING.md:168]: @Tomiiwa Great work! i have one small thing i would like for you to add

this [link ](https://aqicn.org/api/) is great and it contains all APIs used by ozone , and itself contains reference to the JSON API among other APIs 

can u add a link to the JSON API below it?

Comment (Tomiiwa) [reply CONTRIBUTING.md:168]: alright, will do

Comment (Sam-damn) [reply CONTRIBUTING.md:168]: and if you could change the description of the second link to reflect the idea that it offers multiple APIs and not one and thank u very much :)

**Comment by Sam-damn** [comment]
@Tomiiwa Good work ! I will merge this in right now

**Review by Sam-damn — APPROVED** [review]

**Pull request merged**